File: ingestion/config/sources/hackernews.py
from scrapers.hackernews_scraper import scrape_hacker_news


hackernews_source = {
    "slug": "hacker-news",
    "name": "Hacker News",
    "website_url": "https://news.ycombinator.com",
    "type": "WEBSITE",
    "political_lean": "UNKNOWN",
    "reliability_score": 0.70,
    "enabled": True,
    "scraper": scrape_hacker_news
}

File: backend/config/sources/hackerNews.js
const {
    scrapeHackerNews
  } = require("../../scrapers/hackerNewsScraper");
  
  const hackerNewsSource = {
    slug: "hacker-news",
  
    name: "Hacker News",
  
    websiteUrl:
      "https://news.ycombinator.com",
  
    type: "WEBSITE",
  
    scraper: scrapeHackerNews
  };
  
  module.exports = hackerNewsSource;